Rozwinięcie komend

eRgo35 opened the pull request.

eRgo35 committed:
Rozwinięcie komend

VectorKappa commented on main.py:
````diff
@@ -22,10 +24,11 @@
 wybor_klasy = "Witamy na szkolnym serwerze ZSPM! Podaj nam klasę, do której uczęszczasz: \n1ATPDPI\n1ATGPC\n1BTGPC\n1CTI\n1DTI\n1DTM\n1ETGPC\n1FTI\n1FTM\n1GTI\n2ATPD\n2ATGPC\n2CTI\n2DTI\n2DTM\n3ATPD\n3ATGPC\n3BTGPC\n3CTI\n3DTI\n3DTM\n4ATPD\n4ATM\n4BTCPG\n4CTI\n4DTI"
 ostrzezenie_autoryzacji = "Próbowałeś, ale nie. Incydent zgłoszono. \n Chyba że naprawdę jesteś nauczycielem. Wtedy napisz do kogoś z administracji."
 blad_klasy = "Podaj klasę z listy!"
+dekalog = "```1 .   Dołączyć do serwera mogą tylko i wyłącznie uczniowie, nauczyciele oraz absolwenci naszej szkoły. Aby zweryfikować swoją tożsamość, odpowiedz na prywatną wiadomość wysłaną przez @PoliBot\n2.   Na serwerze obowiązuje kultura. Prosimy o zwracanie się do siebie z szacunkiem.\n3.   Zakaz spamowania.\n4.   Zakaz reklamowania.\n5.   Zakaz używania i zapraszania klienckich botów na se ...
````
Nie potrzebujemy zasad, są już na serwerze.

eRgo35 committed:
Drobne poprawki

- Próba ogarnięcia komendy zmien_nick
- Poprawienie błędów związanych z backslashami
- Poprawienie komendy $komendy - lecz wciąż do rozwinięcia
- Usunięcie komend z względu na brak celu egzystencji tej komendy

eRgo35 committed:
Fixing github branches #1
